src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx

```diff
@@ -0,0 +1,106 @@
+import { FormPanel } from '@components/interface';
+import { IconLabel, InputTokenLabel, MaskedIntegerField } from '@components/composite';
+import { Box } from '@mui/system';
+import { Panel, Typography } from "src/components/atomic";
+import { AugmentedMellowLpVault, formatCurrency, toUSFormat } from '@utilities';
+import { Button } from '@components/atomic';
+import { colors } from '@theme';
+import { ReactNode } from 'react';
+import { isUndefined } from 'lodash';
+
+export type LPMellowVaultDepositWindowProps = {
+    lpVault: AugmentedMellowLpVault;
+    onChangeDeposit: (value: number | undefined) => void;
+    submitText: string;
+    hintText: ReactNode;
+    onSubmit: () => void;
+    disabled: boolean;
+};
+
+const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWindowProps> = ({ lpVault, onChangeDeposit, submitText, hintText, disabled, onSubmit }: LPMellowVaultDepositWindowProps) => {
+
+    const subtext = `WALLET BALANCE: ${isUndefined(lpVault.userWalletBalance) ? "---" : `${formatCurrency(lpVault.userWalletBalance, true)} ${lpVault.tokenName}`}`;
+
+    const handleChange = (newValue: string | undefined) => {
+        const usFormatted = toUSFormat(newValue);
+        onChangeDeposit(!isUndefined(usFormatted) ? parseFloat(usFormatted) : undefined);
+    };
+    
+    const renderContent = () => {
```

Comment:
this render content pattern is not really needed here. Just return the component in return

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/commit/a44c3c48e2a2660e1fc14f6eeeda646152d2edea